framework/api/service_collection_builder.py

```diff
@@ -31,16 +31,16 @@ def build_service_provider(self):
 
     def register_api_presenters(self):
         _PresenterClasses = []
-
-        for _Root, _Directories, _Files in os.walk("framework/api/routes"):
+        
+        for _Root, _Directories, _Files in os.walk(os.path.normpath("framework/api/routes")):
 
             DIR_EXCLUSIONS = [r"__pycache__"]
             FILE_EXCLUSIONS = [r".*__init__\.py", r"^.*(?<!\.py)$"]
             Common.apply_exclusion_filter(_Directories, DIR_EXCLUSIONS)
             Common.apply_exclusion_filter(_Files, FILE_EXCLUSIONS)
-
+            
```

Comment:
seems you're leaving whitespace, there's extensions for removing on save (or VSC settings) and for highlighting

Reply:
I have incorporated the following settings in my settings.json file & removed the white spaces referred to by this comment. Hopefully that should reduce/remove this issue.

    "editor.renderWhitespace": "all",
    "files.trimTrailingWhitespace": true